_overviews/scala3-migration/tooling-syntax-rewriting.md

```diff
@@ -126,6 +138,8 @@ After this first rewrite, we can use the significant indentation syntax to remov
 To do that we use the `-indent` option in combination with the `-rewrite` option.
 It leads us to the following version:
 
+{% tabs scala-3-location_2 %}
```

Comment:
this id was duplicated, which breaks the site rendering

```suggestion
{% tabs scala-3-location_2b %}
```

Reply:
Note for self: perhaps its better to generate unique ids in the Jekyll tabs plugin, as there is not a possibility to validate uniqueness